advanced/workflows/ancient-egyptian-email.mdx

```diff
@@ -0,0 +1,176 @@
+    shinkaiLlmPromptProcessor,
+    shinkaiSqliteQueryExecutor,
+} from './shinkai-local-tools.ts';
+```
+
+### Core Components
+
+The assistant consists of four main operations:
+
+1. **Reading New Emails**
+```typescript
+let { emails, login_status } = await emailFetcher({
+    from_date: inputs.from_date || '',
+    to_date: inputs.to_date || '',
+});
+```
+
+2. **Checking for Previously Answered Emails**
+```typescript
+// Use parameterized queries for security
+const answeredEmailsQuery = await shinkaiSqliteQueryExecutor({
+    query: 'SELECT * FROM answered_emails WHERE processed = ?',
+    params: [false]
+});
+const answeredEmails = (answeredEmailsQuery.result as ANSWERED_EMAIL_REGISTER[]) ?? [];
+```
+
+3. **Generating Responses with LLM**
+```typescript
+const response = await shinkaiLlmPromptProcessor({
+    format: 'text',
+    prompt: `You are a helpful email answering system.
+    Please respond to a following email based on the provided context:
+    <context>
+      ${config.response_context}
+    </context>
+    This is the email you need to respond to:
+    <email>
+      <email.subject>${email.subject}</email.subject>
+      <email.sender>${email.sender}</email.sender>
+      <email.date>${email.date}</email.date>
+      <email.text>${email.text}</email.text>
+    </email>`,
+});
+```
+
+4. **Sending Responses**
+```typescript
+await sendEmail({
+    recipient_email: email.sender,
+    subject: 'RE:' + email.subject,
+    body: response.message
+});
+```
+
+## Tool Configuration
+
+The tool's behavior is defined by its metadata. Here are the key configuration fields:
+
+```json
+{
+  "configurations": {
+    "properties": {
+      "response_context": {
+        "type": "string",
+        "description": "The context to guide the email responses"
+      }
+    }
+  },
+  "parameters": {
+    "properties": {
+      "from_date": {
+        "type": "string",
+        "description": "The starting date for fetching emails in DD-Mon-YYYY format"
+      },
+      "to_date": {
+        "type": "string",
+        "description": "The ending date for fetching emails in DD-Mon-YYYY format"
+      }
+    }
+  }
+}
+```
+
+### Ancient Egyptian Personality
+
+Configure the assistant's personality through the response context:
+
+```typescript
+const config = {
+    response_context: `
+    You are an ancient pharaoh mummy named Khufur that's been trapped 
+    for thousands of years in your cursed sarcophagus. You've been 
+    tasked with responding to emails but take every opportunity to 
+    remind everyone you are trapped in your ancient tomb and you'll 
+    curse the whole world once you get out.
+    `
+};
+```
+
+## Testing the Tool
+
+1. In the Tool Editor's run section:
+   - Fill in the configuration fields
+   - Set date parameters if needed
+   - Click Run to test
+
+2. Test through chat:
+   - Start a new chat
+   - Use the tool with a command like: "Please answer emails from today"
+
+3. Set up scheduled execution:
```

Comment:
Explain more about the Scheduled Tasks section. How do we navigate to it, what do we encounter, what button do we click to create a new scheudled task, what does each of the fields of the seciton does, etc